src/main/java/pl/anasz/cinema/enities/Room.java

```diff
@@ -0,0 +1,30 @@
+package pl.anasz.cinema.enities;
+
+import lombok.Data;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.ManyToOne;
+import javax.persistence.JoinColumn;
+import javax.persistence.Table;
+import javax.persistence.Id;
+
+@Entity
+@Data
+@Table(name = "rooms")
+public class Room {
+
+    @Column(name = "room_id")
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @Column(name = "room_number")
+    private long number;
```

Comment:
Brak adnotacji `@NotNull` tutaj i w kilku innych encjach. Wiesz czemu trzeba to ustawić?

Reply:
Oczywiście, należałoby ustawić @NotNull aby uniknąć NullPointerException, dodatkowo taka informacja do użytkownika dawałaby jasno do zrozumienia, że zapomniał uzupełnić któregoś pola.

Reply:
Właściwie jest to moja pomyła, z której wyszło pytanie-pułapka ;). Ten `@NotNull` nie jest tu potrzebny, skoro używamy "małego" longa.

Reply:
Racja! A ja utrzymałam się tego, że wiem, że w niektórych miejscach mogłabym dodać tę adnotację, a nie zwróciłam uwagi, przy którym polu został pozostawiony ten komentarz...